CONTRIBUTING.md

```diff
@@ -14,9 +14,8 @@ contribution. See the [DCO](DCO) file for details.
 
 # Email and Chat
 
-The project uses the the cni-dev email list and #appc on Freenode for chat:
+The project uses the the cni-dev email list:
 - Email: [cni-dev](https://groups.google.com/forum/#!forum/cni-dev)
-- IRC: #[appc](irc://irc.freenode.org:6667/#appc) IRC channel on freenode.org
```

Comment:
Personally I'd prefer #cni but someone must've recently joined that channel and I think we have to wait until that event times out. If said someone is reading this by any chance, are you willing to hand #cni over?